.github/workflows/node.yml

```diff
@@ -39,6 +36,18 @@ jobs:
             eslint: 8
           - node-version: 10
             eslint: 8
+          - node-version: 15
+            eslint: 9  
+          - node-version: 14
+            eslint: 9
+          - node-version: 13
+            eslint: 9
+          - node-version: 12
+            eslint: 9
```

Comment:
ESLint v9 seems to import modules with node:fs. Therefore, I exclude that because it would be an error for Node.14 or lower.

![image](https://github.com/user-attachments/assets/38f4506d-1c79-4961-99a5-f68443a5f6d2)

Reply:
eslint 9 should be skipped on any node outside of the `^18.18.0 || ^20.9.0 || >=21.1.0` range, because that's what its `engines.node` declares.

Reply:
Sorry, i fixed